### PR TITLE
runtime configurable chain id

### PR DESCRIPTION
**Pull request by mateuszaaa**
- **Using debug print for call context**
- **configurable chain id**
- **align with new format of Message struct**

### CODE REVIEW

File: pallets/metamask-signature/src/benchmarking.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//! Benchmarking setup for pallet-template
+#![cfg(feature = "runtime-benchmarks")]
+use super::*;
+
+#[allow(unused)]
+use crate::Pallet as Template;
```
Comment (iStrike7) [inline pallets/metamask-signature/src/benchmarking.rs]: Do we really want these template module stuff here? benches, mock and tests...?

File: pallets/metamask-signature/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+						"name": "",
+						"version": "",
+						"chainId": ""
+					},
+					"message": {
+						"call": "",
+						"tx": ""
+					}
+			}"#;
+			log::info!(target: "xxx", "hello world 88 ");
```
Comment (iStrike7) [inline pallets/metamask-signature/src/lib.rs]: Lot's of these debug logs around

File: rollup/node/src/chain_spec.rs [hunk 3]
```diff
@@ -321,5 +323,11 @@ fn rollup_genesis(
 			},
 		},
 		rolldown: rollup_runtime::RolldownConfig { _phantom: Default::default() },
+		metamask: rollup_runtime::MetamaskConfig {
+			name: "Gasp".to_string(),
+			version: "0.0.1".to_string(),
+			chain_id: 17000u64,
```
Comment (iStrike7) [inline rollup/node/src/chain_spec.rs]: Don't you want to use eth_chain_id here?

**Review by iStrike7 — APPROVED** [review]
Some minor typo/cleanup comments

File: pallets/metamask-signature/src/benchmarking.rs (hunk 1 above)
Comment (mateuszaaa) [reply pallets/metamask-signature/src/benchmarking.rs]: actually we dont need them :+1:

File: rollup/node/src/chain_spec.rs (hunk 3 above)
Comment (mateuszaaa) [reply rollup/node/src/chain_spec.rs]: :man_facepalming:

**Pull request merged**